.pre-commit-config.yaml

```diff
@@ -22,3 +22,9 @@ repos:
       - id: trailing-whitespace
       - id: mixed-line-ending
       - id: sort-simple-yaml
+
+  - repo: https://github.com/gitleaks/gitleaks
+    rev: v8.16.3
```

Comment:
How do we ensure that this rev. (and the other two) are updated when new versions are shipped. The https://github.com/pre-commit/pre-commit-hooks exists in a v5.0.0 today and the mdformat one exists in 7.21.

Reply:
I hoped _dependabot_ will start supporting pre-commit-config, but it seems it's not going to the case in the near future.

There's this tool: https://pre-commit.ci, but it doesn't look like something we want for AL-Go at this point.

Reply:
I bumped the versions manually for now.